### ISSUE TITLE
Level: "UFO"

### ISSUE THREAD

**Issue by tttppp** [andybalaam/rabbit-escape#53]
This was fun to make!  Possibly I've gone too far from the original "underground" series, past "outdoors" and into "space". I'd be interested to know how people solve this - I have a solution in mind, but there might be others too.
```
:name=UFO
:description=
:num_rabbits=20
:num_to_save=18
:dig=1
:bridge=3
:explode=3
:climb=2
:rabbit_delay=4
    #            #        #
          #          #     
 #    #        #           
                           
                           
                           
                           
                           
            /#\            
       / /### ###\ \       
      / (    #    ) \      
     / / ##\ O /## \ \     
   #( \  #Q#####Q#  / )#   
    #\ \ #   #   # / /#    
     #\  ### # ###  /#     
    / #\           /# \    
   /#  #############  #\   
```

**Comment by colonelfazackerley** [comment]
I stared at it meditatively for 10 minutes after all my clockwise rabbits died on the first attempt.

Did it 3rd time (all rabbits escaped). Very enjoyable.

**Comment by tttppp** [comment]
Ah, I now see a way to save all rabbits. That wasn't the intended method, but I'm not sure if I can prevent it. Maybe I'll have a go creating another UFO level that requires my solution. I think this one is good with a choice of solutions.

**Comment by andybalaam** [comment]
Just a note: I remain increasingly excited about all these new levels and code, but am currently unable to keep pace with you @colonelfazackerley and @tttppp.  Never fear though - these levels will be included in a release soon.

**Comment by tttppp** [comment]
Another solution to worry about:

![screenshot_2015-09-25-18-16-40](https://cloud.githubusercontent.com/assets/7073263/10108053/a9b821a8-63b6-11e5-9f6d-ac4274abcfca.png)

**Comment by andybalaam** [comment]
The counter rotational dynamics of this level are breathtaking. I think it's my favourite so far.

**Comment by andybalaam** [comment]
I also saved all of them but not until I'd watched one group of rabbits circulate for about 5 brilliant minutes.

**Comment by colonelfazackerley** [comment]
close me
merged to master with f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a

**Comment by andybalaam** [comment]
The second hint has an extraneous backslash (in the snapshot I am using. ) Feel free to close if this is already fixed.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721
